examples/camera/README.txt

```diff
@@ -3,31 +3,40 @@ examples/camera
 
   This sample code gets an image from the image sensor.
 
-  Supported image sensor is ISX012.
+  Supported image sensor is ISX012 and will work with the official Spresense camera board.
 
 
-  This example can be used by camera default config.
+  This example can be used with the camera example default configuration
 
-  $ ./tools/config.py camera
+  $ ./tools/config.py examples/camera
 
-  This example can be output captured images to LCD directly.
-  If you want to work with LCD devices, type like this:
+  This example can also output captured images to an LCD directly.
+  If you want to work with LCD devices, configure like this:
 
-  $ ./tools/config.py camera ili9340
+  $ ./tools/config.py examples/camera device/lcd
 
-  And enable following options.
+  And enable following option in /sdk/configs/device/camera-defconfig
 
-  CONFIG_EXAMPLES_CAMERA_OUTPUT_LCD -- Show captured image on the LCD
-  CONFIG_EXAMPLES_CAMERA_INFINITE   -- Capture infinitely
+  CONFIG_EXAMPLES_CAMERA_OUTPUT_LCD=y
+  
+  Currently supports ili9340 and lpm013m091a based displays.
+  
+  Default resolution is 320x240.  
+  This can be modified in the LCD driver code at /sdk/drivers/lcd/ili9340.c
 
+  The LCD is expected to be connected on SPI4 (available on the Spresense expansion board)
```

Comment:
Hi @toolboc 

Great that you are contributing to the Spresense project. Thank you!
I created an Kconfig option so to make the display connection selectable,  SPI4 extension board or SPI5 main board.
You can see this change in: 6f4957b861357756e868b759e15e6e1fe2a4a8a0
Just a comment, could be worth mentioning.
BR

Reply:
@TE-KarlKomierowski , your PR and Hackster article gave enough clues to allow me to successfully get the camera demo to work.  Thanks for that!  

I've made a note of the configuration now that your changes have been accepted.